src/com/google/javascript/jscomp/AbstractCommandLineRunner.java

```diff
@@ -292,12 +291,6 @@ static DependencyOptions createDependencyOptions(
           .setDependencySorting(true)
           .setMoocherDropping(true)
           .setEntryPoints(closureEntryPoints);
-    } else if (processCommonJSModules) {
-      return new DependencyOptions()
-        .setDependencyPruning(false)
-        .setDependencySorting(true)
-        .setMoocherDropping(false)
-        .setEntryPoints(closureEntryPoints);
```

Comment:
Could you update `DependencyOptions` and document the new meaning of `EntryPoints`?